bsdlib/include/nrf_socket.h

```diff
@@ -184,15 +185,36 @@ typedef int32_t ssize_t;
 #define NRF_GNSS_SV_FLAG_UNHEALTHY    8 /**< Indicate that the satellite is unhealthy. */
 /**@} */
 
+/**@defgroup nrf_gnss_agps_data_type Flags indicating which AGPS assistance data set is written to the GPS module
+ * @ingroup nrf_socket
+ * @{
+ */
+#define NRF_GNSS_AGPS_UTC_PARAMETERS                   1 /**< GPS UTC assistance AGPS parameters. */
+#define NRF_GNSS_AGPS_EPHEMERIDES                      2 /**< GPS ephemeris assistance AGPS parameters. */
+#define NRF_GNSS_AGPS_ALMANAC                          3 /**< GPS almanac assistance AGPS parameters. */
+#define NRF_GNSS_AGPS_KLOBUCHAR_IONOSPHERIC_CORRECTION 4 /**< GPS ionospheric assistance AGPS parameters, Klobuchar model. */
+#define NRF_GNSS_AGPS_NEQUICK_IONOSPHERIC_CORRECTION   5 /**< GPS ionospheric assistance AGPS parameters, NeQuick model. */
+#define NRF_GNSS_AGPS_GPS_SYSTEM_CLOCK_AND_TOWS        6 /**< GPS system time and SV TOW assistance AGPS parameter. */
+/** @} */
+
 /**@defgroup nrf_nmea_str_mask Set of values (as bitmask) to enable NMEA output strings
  * @ingroup nrf_socket
  * @{
  */
-#define NRF_CONFIG_NMEA_GGA_MASK     1  /**< Enables Global Positioning System Fix Data. */
-#define NRF_CONFIG_NMEA_GLL_MASK     2  /**< Enables Geographic Position Latitude/Longitude and time. */
-#define NRF_CONFIG_NMEA_GSA_MASK     4  /**< Enables DOP and active satellites. */
-#define NRF_CONFIG_NMEA_GSV_MASK     8  /**< Enables Satellites in view. */
-#define NRF_CONFIG_NMEA_RMC_MASK     16 /**< Enables Recommended minimum specific GPS/Transit data. */
+#define NRF_GNSS_NMEA_GGA_MASK     1  /**< Enables Global Positioning System Fix Data. */
+#define NRF_GNSS_NMEA_GLL_MASK     2  /**< Enables Geographic Position Latitude/Longitude and time. */
+#define NRF_GNSS_NMEA_GSA_MASK     4  /**< Enables DOP and active satellites. */
+#define NRF_GNSS_NMEA_GSV_MASK     8  /**< Enables Satellites in view. */
+#define NRF_GNSS_NMEA_RMC_MASK     16 /**< Enables Recommended minimum specific GPS/Transit data. */
+/** @} */
+
+/**@defgroup nrf_gnss_psm_modes Values indicating the power saving mode used in the GNSS module
```

Comment:
This sentence sounds a bit strange. 'indicating' sounds like the application is not an active part of this data.

Reply:
```suggestion
/**@defgroup nrf_gnss_psm_modes Values setting the power saving mode used in the GNSS module
```

Does that make more sense?